pentaho-js/src/main/javascript/cdf/components/ccc/config/cdf.vizApi.conf.js

```diff
@@ -21,6 +21,22 @@ define([], function() {
         priority: -1,
         select: {
           application: "pentaho-cdf",
+          type: [
+            "pentaho/ccc/visual/heatgrid",
```

Comment:
This margin for heatgrid is not specific to cdf. We'll do it in common-ui instead.